### PR TITLE
chore: minutes for 2024-11-20

### PR DESCRIPTION
**Pull request by marco-ippolito**
Fixes: https://github.com/nodejs/typescript/issues/18

@nodejs/typescript

### CODE REVIEW

File: meetings/2024-11-20.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+### nodejs/typescript
+
+* Type Stripping in `node_modules/*/*.ts` [#14](https://github.com/nodejs/typescript/issues/14)
+    Marco:  No need to support it for the unflagging, it can wait.
+    Daniel: Right now don’t think it’s a good idea, but not definitive.
+    Jacob: use engines section of package.json to mitigate some the incompatibility issues
+    Marco: probably enable it behind a flag
+    Daniel: monorepo use symlinks so don’t need it
+    Andrew: monorepo already work, maybe does not work when publishing .ts files internally
+    Daniel: if users wants to use workspaces, that is already documented
```
Comment (AugustinMauroy) [inline meetings/2024-11-20.md:38]:
```suggestion
    Daniel: If users wants to use workspaces, that is already documented
```

File: meetings/2024-11-20.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    Jacob: monorepo in fact works, tested in codemod
+    Marco: We believe monorepo use-cases should all work, we will re-evaluate after the feature becomes more stable.
+
+* Unflag `--experimental-strip-types` [#17](https://github.com/nodejs/typescript/issues/17)
+    Jacob: volunteers to document how to publish a typescript package on npm
```
Comment (AugustinMauroy) [inline meetings/2024-11-20.md:43]:
```suggestion
    Jacob: Is volunteers to document how to publish a typescript package on npm
```

File: meetings/2024-11-20.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    Daniel: ts 5.7 landing tomorrow
+    Daniel: About the ack from TypeScript team, exciting, the missing piece is providing errors in the IDE but no blockers.
+    Marco: transform types might always stay behind a flag, source maps should not be enabled by default. Marco should we wait to unflag, for a TS compatible flag?
+    Daniel: Not a big deal, it’s ok
+    Conclusion: TS team are ok with unflagging
+    TS Issue for option to disallow features requiring transformations which are not supported by Node.js' --strip-types:  <https://github.com/microsoft/TypeScript/issues/59601>
+    Andrew/Daniel:  The plan is for TS to add the lint flag in TS 5.8 (the next release)
+    Marco: documentation was added: <https://nodejs.org/docs/latest/api/typescript.html#type-stripping>
+    Marco: talking about next steps, should we ping frontend frameworks?
+    Daniel: would be good but not necessary
```
Comment (AugustinMauroy) [inline meetings/2024-11-20.md:53]:
```suggestion
    Marco: Documentation was added: <https://nodejs.org/docs/latest/api/typescript.html#type-stripping>
    Marco: Talking about next steps, should we ping frontend frameworks?
    Daniel: Would be good but not necessary
```

**Review by AugustinMauroy — APPROVED** [review]

**Comment by JakobJingleheimer** [comment]
Closes https://github.com/nodejs/typescript/issues/18

File: meetings/2024-11-20.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+* Type Stripping in `node_modules/*/*.ts` [#14](https://github.com/nodejs/typescript/issues/14)
+    Marco:  No need to support it for the unflagging, it can wait.
+    Daniel: Right now don’t think it’s a good idea, but not definitive.
+    Jacob: use engines section of package.json to mitigate some the incompatibility issues
+    Marco: probably enable it behind a flag
+    Daniel: monorepo use symlinks so don’t need it
+    Andrew: monorepo already work, maybe does not work when publishing .ts files internally
+    Daniel: If users wants to use workspaces, that is already documented
+    Jacob: monorepo in fact works, tested in codemod
+    Marco: We believe monorepo use-cases should all work, we will re-evaluate after the feature becomes more stable.
```
Comment (JakobJingleheimer) [inline meetings/2024-11-20.md:40]: This does not format correctly in markdown (it collapses into a single continuous paragraph), but I don't know how to get it to do so.

File: meetings/2024-11-20.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+* Unflag `--experimental-strip-types` [#17](https://github.com/nodejs/typescript/issues/17)
+    Jacob: Is volunteers to document how to publish a typescript package on npm
```
Comment (JakobJingleheimer) [inline meetings/2024-11-20.md:43]:
```suggestion
    Jacob: volunteers to document how to publish a typescript package on npm
```

File: meetings/2024-11-20.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    Daniel: ts 5.7 landing tomorrow
+    Daniel: About the ack from TypeScript team, exciting, the missing piece is providing errors in the IDE but no blockers.
+    Marco: transform types might always stay behind a flag, source maps should not be enabled by default. Marco should we wait to unflag, for a TS compatible flag?
+    Daniel: Not a big deal, it’s ok
+    Conclusion: TS team are ok with unflagging
+    TS Issue for option to disallow features requiring transformations which are not supported by Node.js' --strip-types:  <https://github.com/microsoft/TypeScript/issues/59601>
+    Andrew/Daniel:  The plan is for TS to add the lint flag in TS 5.8 (the next release)
+    Marco: Documentation was added: <https://nodejs.org/docs/latest/api/typescript.html#type-stripping>
+    Marco: Talking about next steps, should we ping frontend frameworks?
+    Daniel: Would be good but not necessary
```
Comment (JakobJingleheimer) [inline meetings/2024-11-20.md:53]: Same here: collapses into a single paragraph

File: meetings/2024-11-20.md (hunk 5 above)
Comment (AugustinMauroy) [reply meetings/2024-11-20.md:43]:
```suggestion
    Jacob: Volunteers to document how to publish a typescript package on npm
```

Just to normalize with the rest of the doc

**Review by JakobJingleheimer — APPROVED** [review]
🙌

**Comment by marco-ippolito** [comment]
@JakobJingleheimer why your approval is gray? 🤔

**Comment by JakobJingleheimer** [comment]
I think I have only read-write permissions on the repo (I think anything lower than `maintainer` doesn't count).

**Pull request merged**